### PR TITLE
Add example for simulating multiple diseases with DummyModel.

### PR DESCRIPTION
**Pull request by jamesturner246**
This PR adds a simple example using the `DummyModel` for simulating many 'problematic' diseases, rather than just IHD.

Addresses https://github.com/imperialCHEPI/healthgps/issues/320.

### CODE REVIEW

File: example_diseases/data/analysis/disability_weights.csv
Comment (jzhu20) [inline example_diseases/data/analysis/disability_weights.csv:1]: Do we need to add disease weight of the other four diseases?

File: example_diseases/Dummy.json
Comment (jzhu20) [inline example_diseases/Dummy.json:1]: Is it better to use a similar level of sodium as in India, in order to apply a similar relative risk? For example, value is 4000 and policy effect is -2000.

File: example_diseases/data/analysis/disability_weights.csv
Comment (jamesturner246) [reply example_diseases/data/analysis/disability_weights.csv:1]: For this 'minimal' example, I'm only testing the risk factor relative weights. Those disease weights will also need looking at, but the factor relative weights need testing in isolation.

File: example_diseases/Dummy.json
Comment (jamesturner246) [reply example_diseases/Dummy.json:1]: In these tests, I have exaggerated values to test that they are actually working in the first place. But yeah, we can modify them once we know they're working.

**Comment by jamesturner246** [comment]
Thanks for the review! If you're happy, could you make it an 'approve' review if you're happy with the changes? Or use 'request changes' if you're not. I can't merge it until There's an 'approve' review. Thanks!

**Review by jzhu20 — APPROVED** [review]

**Pull request merged**
